.github/PULL_REQUEST_TEMPLATE/control_plane_template.md

```diff
@@ -1,49 +1,46 @@
+and followed the instructions by checking all the boxes:
 
-Otherwise your PR may be subject to ARM review requirements. Complete the following:
-- [ ] Check this box if any of the following apply to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
-  - Adding a new service
-  - Adding new API(s)
-  - Adding a new API version
-    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version). Note that this doesn't apply if you are trying to merge a PR that was previously in the private repository.
+- [ ] I confirm this PR is modifying Azure Resource Manager (ARM) related specifications, and not data-plane related specifications.
+- [ ] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy).
+- [ ] If I am adding a new service, API, or API version, I have ensured to copy the existing version into a new
+  directory structure for first commit and then I pushed new changes, including version updates, in separate commits.  
+  - I understand I can use [OpenAPI Hub](https://aka.ms/openapihub) to initialize the PR for adding a new version.  
+  - I understand that this doesn't apply if this PR was previously a PR in the [private repository](https://github.com/Azure/azure-rest-api-specs-pr).
+- [ ] I have reviewed following [guidelines](https://aka.ms/rpguidelines), including
+  [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and
+  [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md) (estimated time: 4 hours).  
+  I understand this is required before I can request review from an ARM API Review board.
 
-- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.
+### ARM API changes review
 
-- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them.
+- If you want for the ARM team to review this PR, you must add the `ARMReview` label. 
+- The automation may automatically add the `ARMReview` label, if appropriate.
```

Comment:
>appropriate [](http://example.com/codeflow?start=65&length=11)

appropriate, in which case, there is no action required from your side.

Reply:
@rkmanda the automation may decide to add `ARMChangesRequested` label which will mean there is in fact some action required from the PR author.

Reply:
I added:

> If this happens, proceed according to guidance given in GitHub comments also added by the automation.